t/t0024-content-s3.t

```diff
@@ -139,6 +163,73 @@ test_expect_success HAVE_JQ 'kvs-checkpoint.get foo returns same value' '
         test_cmp value2.exp value2.out
 '
 
+test_expect_success 'config: reload config does not take effect immediately' '
+	flux config reload 2>/dev/null &&
+	flux dmesg | grep content-s3 | tail -1 >reload.log &&
+	grep "changes will not take effect until next flux restart" reload.log
+'
+
+test_expect_success 'save good config' '
+	mv content-s3.toml content-s3.save &&
+	mv creds/creds.toml creds/creds.save
+'
+
+test_expect_success 'config: reload with extra field fails' '
+	cp content-s3.save content-s3.toml &&
+	echo "extrafield = \"failure\"" >>content-s3.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: reload with malformed uri fails' '
+	cp content-s3.save content-s3.toml &&
+	sed -i -e "s/uri =.*$/uri = \"baduri\"/" \
+		content-s3.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: reload with bad credential path fails' '
+	cp content-s3.save content-s3.toml &&
+	sed -i -e "s/credential-file =.*$/credential-file = \"nocreds\"/" \
+		content-s3.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: reload with extra s3 key fails' '
+	cp creds/creds.save creds/creds.toml &&
+	echo "extrakey = \"failure\"" >>creds/creds.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: unload module' '
+	flux module remove content-s3
+'
+
+test_expect_success 'config: module fails to load without config file' '
+	rm -f content-s3.toml &&
+	cp creds/creds.save creds/creds.toml &&
+	test_must_fail flux config reload
+	flux config reload &&
+	test_must_fail flux module load content-s3
+'
+
+test_expect_success 'config: module fails to load without credentials file' '
+	rm -f creds/creds.toml &&
+	cp content-s3.save content-s3.toml &&
+	test_must_fail flux config reload
+	flux config reload &&
+	test_must_fail flux module load content-s3
+'
+
+test_expect_success 'config: restore good config' '
+	mv -f content-s3.save content-s3.toml &&
+	mv -f creds/creds.save creds/creds.toml &&
+	flux config reload
+'
+
+test_expect_success 'config: load module' '
+	flux module load content-s3 testing
```

Comment:
Oopps the "testing" option is a cut-and-paste-o from the content-files test and has no meaning here.  I see it in a couple other places too, please remove.